src/bin/tsservices_dev.js

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line node/no-unpublished-import
+import * as ts from "typescript";
+import path from 'path';
+import fs from 'fs';
+
+const LIBDTS = `// Standard types
+interface Date {}
+
+type Record<K extends keyof any, T> = {
+	[P in K]: T;
+};
+`;
```

Comment:
we could read this with `fs.redFileSync` from `src/node_modules/app/tsservices/LIBD.ts` to avoid duplication`